metrics/http_middleware.lua

```diff
@@ -103,6 +104,7 @@ end
 -- @return middleware
 -- @usage router:use(http_middleware.v2(), {name = 'http_instrumentation'})
 function export.v2(collector)
+    log.warn('HTTP v2 is deprecated and will be removed in next releases. Use HTTP v1 instead')
```

Comment:
You can't write in log on each request. It should be done only at first request to avoid unexpected (for user) log flooding.

Reply:
See https://github.com/tarantool/errors/blob/master/errors/deprecate.lua or https://github.com/tarantool/vshard/commit/b035fd44fd02b0feb22cb313a208328026ec8d3e#diff-85f9e399cd9ae941a0d5d57372aa49892d5e2798364c02ddc335f6af08060709R1066